src/ActionsImporter/Commands/ContainerCommand.cs

```diff
@@ -10,7 +10,8 @@ public abstract class ContainerCommand : BaseCommand
 
     protected ContainerCommand(string[] args)
     {
-        _args = args;
+        // Don't forward the --prerelease flag to GitHub Actions Importer image
+        _args = args.Where(arg => !arg.Contains(Common.Prerelease.Name, StringComparison.Ordinal)).ToArray();
```

Comment:
@ethanis I'm thinking that removing the prerelease argument from any command passed to the container is fine for now.

If we find ourselves implementing more arguments that are handled only by `gh-actions-importer`, then we could extend this.

Lmk if you can think of a better way to do this!

Reply:
This works for me!